src/cashier_frontend_new/vite.config.ts

```diff
@@ -21,6 +21,14 @@ export default defineConfig({
       outdir: "./src/lib/paraglide",
     }),
   ],
+  optimizeDeps: {
+    esbuildOptions: {
+      define: {
+        // polyfill for esbuild, use for any nodejs standard library module used in the browser
```

Comment:
This is a bad practice that can lead to runtime errors. If this unavoidable, is there a way to restrict it only to the HttpAgent package?